### PR TITLE
[el_vietnamese] Vietnamese Keyboard

### PR DESCRIPTION
**Pull request by MakaraSok**

### CODE REVIEW

**Comment by DavidLRowe** [comment]
@mcdurdin I think my additions were added to Makara's PR. Could you review it and merge it if it looks good? Thanks!

**Comment by DavidLRowe** [comment]
A comment in the .kmn file has "OR Vietnamese 1.0" whereas README.md and HISTORY.md have version 1.1. The comment in the .kmn file should be updated (or deleted).

A comment in the .kmn file has "andrewc AT vicnet.net.au" whereas README.md has <lang.support@gmail.com>as the contact. Which is correct?

Other than those minor changes, everything looks good.

**Comment by MakaraSok** [comment]
@DavidLRowe Since they are comments, I think it should be safe delete them altogether.

**Comment by mcdurdin** [comment]
Versioning questions need to be resolved before we merge this keyboard. There may be an existing newer binary version.

File: release/el/el_vietnamese/source/el_vietnamese.kps [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package>
```
Comment (mcdurdin) [inline release/el/el_vietnamese/source/el_vietnamese.kps]: We should add a `<Keyboards>` section and appropriate language codes.

File: release/el/el_vietnamese/source/el_vietnamese.kps [hunk 2]
```diff
@@ -70,5 +46,15 @@
       <FileType>.png</FileType>
     </File>
   </Files>
+  <Keyboards>
+    <Keyboard>
+      <Name>OR Vietnamese</Name>
+      <ID>el_vietnamese</ID>
+      <Version>1.1</Version>
+      <Languages>
+        <Language ID="vi">vi</Language>
```
Comment (mcdurdin) [inline release/el/el_vietnamese/source/el_vietnamese.kps]: The language name should be "Vietnamese", not "vi" (ID is correct).

**Pull request closed without merging**